npm/npm.go

```diff
@@ -188,6 +189,12 @@ func NewNetworkPolicyManager(clientset *kubernetes.Clientset, informerFactory in
 	iptMgr := iptm.NewIptablesManager()
 	iptMgr.UninitNpmChains()
 
+	log.Logf("Azure-NPM creating, cleaning existing Azure NPM IPSets")
+	destroyErr := ipsm.NewIpsetManager().DestroyNpmIpsets()
+	if destroyErr != nil {
+		log.Logf("Azure-NPM error occurred while destroying existing IPSets err: %s", destroyErr.Error())
```

Comment:
nit, wondering, if DestroyNpmIpsets did metrics.SendErrorMetric inside its function, whether do we want to log here again.